pkg/webreg/webreg.go

```diff
@@ -365,6 +366,252 @@ const templateDefinitions = `
 {{ end }}
 `
 
+const ciOperatorOverviewPage = `<h2 id="title"><a href="#title">What is <code>ci-operator</code> and how does it work?</a></h2>
+
+<p>
+<code>ci-operator</code> is a highly opinionated test workflow execution engine
+that knows about how OpenShift is built, released and installed. <code>ci-operator</code>
+hides the complexity of assembling an ephemeral OpenShift 4.x release payload,
+thereby allowing authors of end-to-end test suites to focus on the content of
+their tests and not the infrastructure required for cluster setup and installation.
+</p>
+
+<p>
+In addition to giving first-class support for testing OpenShift components,
+<code>ci-operator</code> expects to run in an OpenShift cluster and uses
+OpenShift features like <code>Builds</code> and <code>ImageStreams</code>
+extensively, thereby exemplifying a complex OpenShift user workflow and
+making use of the platform itself. Each test with a unique set of inputs
+will have a <code>Namespace</code> provisioned to hold the OpenShift objects
+that implement the test workflow.
+</p>
+
+<p>
+<code>ci-operator</code> needs to understand a few important characteristics of
+any repository it runs tests for. This document will begin by walking through
+those characteristics and how they are exposed in the configuration. With an
+understanding of those building blocks, then, the internal workflow of
+<code>ci-operator</code> will be presented.
+</p>
+
+<h3 id="configuration"><a href="#configuration">Configuring <code>ci-operator</code>: Defining A Repository</a></h3>
+
+<p>
+At a high level, when a repository author writes a <code>ci-operator</code>
+configuration file, they are describing how a repository produces output
+artifacts, how those artifacts fit into the larger OpenShift release and
+how those artifacts should be tested. The following examples will describe
+the configuration file as well as walk through how <code>ci-operator</code>
+creates OpenShift objects to fulfill their intent. 
+</p>
+
+<h4 id="inputs"><a href="#inputs">Configuring Inputs</a></h4>
+
+<p>
+When <code>ci-operator</code> runs tests to verify proposed changes in a pull
+request to a component repository, it must first build the output artifacts
+from the repository. In order to generate these builds, <code>ci-operator</code>
```

Comment:
Should clarify that component repository and repository mean "list of images with unique names that together form a branch or product release" a bit earlier.  Maybe in the second paragraph

> ci-operator helps assemble releases out of multiple container images by maintaining an integration stream of images that can be snapshot to perform pull-request or release testing.  A pull request for a given repository results in a subset of images being rebuilt, while the remaining images are inherited from the current state of the integration stream.  After a pull request merges, those new images are built and pushed into the integration stream for others to consume.  On top of that integration stream, ci-operator also allows a "release image" at a point in time to act as a single artifact to deliver to product consumers.

Or similar.